.github/workflows/linux_ci.yml

```diff
@@ -39,7 +39,6 @@ jobs:
         run: |
           echo $VFLAGS
           make
-          ./v test-cleancode
```

Comment:
No. That is here on purpose, so that it can fail as fast as possible.

It only adds a few seconds to the run, while it prevents much more expensive checks done for code that is not even formatted properly.

Reply:
I see. That's the other option I though why it might be there. But the more expensive checks still run for gcc and clang. Failing here in tcc won't interrupt them. 

If it makes sense to have formatting verification run before the other cc CI I'd submit something here.